collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieVideoActivity.java

```diff
@@ -74,7 +74,7 @@ protected void onCreate(Bundle savedInstanceState) {
         this.camPreview.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (Collect.allowClick()) {
+                if (Collect.allowClick(this.getClass().getSimpleName())) {
```

Comment:
I would recommend using `getName()` not `getSimpleName()`. This method might be called from activity or fragment and it's possible that activity and fragment have the same name. Such a case is not possible in our code but you know just to be sure it's a unique value.